recipes/libxft/all/conanfile.py

```diff
@@ -1,78 +1,86 @@
+import os
+
 from conan import ConanFile
-from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, chdir, rm, rmdir
-from conans import AutoToolsBuildEnvironment
-import functools
+from conan.tools.files import apply_conandata_patches, chdir, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain, PkgConfigDeps
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.53.0"
 
-required_conan_version = ">=1.52.0"
 
 class libxftConan(ConanFile):
     name = "libxft"
-    description = 'X FreeType library'
-    topics = ("libxft", "x11", "xorg")
+    description = "X FreeType library"
+    license = "X11"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.x.org/wiki/"
-    license = "X11"
+    topics = ("x11", "xorg")
 
+    package_type = "library"
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
-    generators = "pkg_config"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
 
     def export_sources(self):
         export_conandata_patches(self)
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
     def requirements(self):
         self.requires("xorg/system")
-        self.requires("freetype/2.13.0")
-        self.requires("fontconfig/2.14.2")
+        self.requires("freetype/2.13.0", transitive_headers=True)
+        self.requires("fontconfig/2.14.2", transitive_headers=True)
```

Comment:
Please try to justify these traits (Specially transitive_libs!) in the future so we don't have to spend extra time looking into the headers :)